features/step_definitions/index_page_steps.rb

```diff
@@ -0,0 +1,7 @@
+Given(/^I am on the index page$/) do
+  visit root_path
+end
+
+Given(/^a restaurant called "([^"]*)" exits$/) do |restaurant_name|
+  FactoryGirl.create(:restaurant, name: restaurant_name)
+end
```

Comment:
There's a new step which was merged from your colleagues. Use that instead (it employs a table) and delete this step.